announcement/announcement-api/api/src/java/org/sakaiproject/announcement/api/AnnouncementService.java

```diff
@@ -123,7 +123,10 @@ public interface AnnouncementService extends MessageService
     public static final String EVENT_ANNC_UPDATE_AVAILABILITY = SECURE_ANNC_ROOT + "revise.availability";
 
     public static final String MOTD_TOOL_ID = "sakai.motd";
-    
+
+	/** SAK-44622 **/
```

Comment:
We don't need the SAK number. We can use the git history to look that up if we need to.

Reply:
done